packages/react-button/src/components/Button/useButtonState.ts

```diff
@@ -32,5 +32,8 @@ export const useButtonState = (draftState: ButtonState) => {
     }
   }
 
-  draftState.disabled = draftState['aria-disabled'] = draftState.disabled || draftState.loading;
+  // TODO: v0 buttons are not disabled while loading, removed `disabled = loading` logic here
```

Comment:
not disabling buttons while loading was intentional - while the UI is loading, it is useful to have the button still focused, because quite often, there is no other suitable place to move focus to.